sstmac/hardware/router/torus_routing.cc

```diff
@@ -107,7 +107,7 @@ class TorusMinimalRouter : public Router {
   void downPath(int dim, int src, int dst, header* hdr) const
   {
     auto& dimensions_ = torus_->dimensions();
-    if (src == ((dst + 1) % dimensions_[dim])){
+    if (src == (dst % dimensions_[dim])){
```

Comment:
same here

Reply:
rightfully so, I don't know the WTH I was looking at. This is absolute nonsense.